ReverseProxy.Auth.Sample doesn't target net6.0

From https://dev.azure.com/dnceng/public/_build/results?buildId=1459222&view=logs&j=0bc77094-9fcd-5c38-f6e4-27d2ae131589&t=e5c9247b-367b-556a-ffed-53eb5f68dbb5&l=175

```
2021-11-08T17:25:27.0292440Z   PDB0021: Document name doesn't match any pattern in Source Link: 'd:\a\_work\1\s\samples\ReverseProxy.Auth.Sample\Views\Account\AccessDenied.cshtml'
2021-11-08T17:25:27.0296058Z   PDB0021: Document name doesn't match any pattern in Source Link: 'd:\a\_work\1\s\samples\ReverseProxy.Auth.Sample\Views\Account\LoggedOut.cshtml'
2021-11-08T17:25:27.0530994Z   PDB0021: Document name doesn't match any pattern in Source Link: 'd:\a\_work\1\s\samples\ReverseProxy.Auth.Sample\Views\Account\Login.cshtml'
2021-11-08T17:25:27.0770501Z D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\7.0.0-beta.21555.2\tools\SymStore.targets(70,5): error MSB3073: The command ""D:\a\_work\1\s\.packages\microsoft.diasymreader.pdb2pdb\1.1.0-beta2-19575-01\tools\Pdb2Pdb.exe" "D:\a\_work\1\s\artifacts\bin\ReverseProxy.Auth.Sample\Release\net6.0\ReverseProxy.Auth.Sample.dll" /out "D:\a\_work\1\s\artifacts\SymStore\Release\ReverseProxy.Auth.Sample\net6.0\ReverseProxy.Auth.Sample.pdb" /srcsvrvar SRC_INDEX=public" exited with code -1. [D:\a\_work\1\s\samples\ReverseProxy.Auth.Sample\ReverseProxy.Auth.Sample.csproj]
2021-11-08T17:25:27.0945924Z ##[error].packages\microsoft.dotnet.arcade.sdk\7.0.0-beta.21555.2\tools\SymStore.targets(70,5): error MSB3073: (NETCORE_ENGINEERING_TELEMETRY=Build) The command ""D:\a\_work\1\s\.packages\microsoft.diasymreader.pdb2pdb\1.1.0-beta2-19575-01\tools\Pdb2Pdb.exe" "D:\a\_work\1\s\artifacts\bin\ReverseProxy.Auth.Sample\Release\net6.0\ReverseProxy.Auth.Sample.dll" /out "D:\a\_work\1\s\artifacts\SymStore\Release\ReverseProxy.Auth.Sample\net6.0\ReverseProxy.Auth.Sample.pdb" /srcsvrvar SRC_INDEX=public" exited with code -1.
```

Originally hit in #972

--- a/samples/Directory.Build.props
+++ b/samples/Directory.Build.props
@@ -4,6 +4,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
 
   <PropertyGroup>
+    <IsShipping>false</IsShipping>
     <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
   </PropertyGroup>
 </Project>

--- a/samples/ReverseProxy.Auth.Sample/ReverseProxy.Auth.Sample.csproj
+++ b/samples/ReverseProxy.Auth.Sample/ReverseProxy.Auth.Sample.csproj
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
     <OutputType>Exe</OutputType>
     <RootNamespace>Yarp.Sample</RootNamespace>
     <LangVersion>latest</LangVersion>

